src/libmw/src/mmr/MMRUtil.cpp

```diff
@@ -93,7 +119,7 @@ BitSet MMRUtil::CalcPrunedParents(const BitSet& unspent_leaf_indices)
     Index last_node = LeafIndex::At(unspent_leaf_indices.size()).GetNodeIndex();
 
     uint64_t height = 1;
-    while ((std::pow(2, height + 1) - 2) <= last_node.GetPosition()) {
+    while ((uint64_t(2) << height) - 2 <= last_node.GetPosition()) {
```

Comment:
There are a couple more places in this file using std::pow(2, ...), maybe we should change those too to be consistent.